src/theme/components/feedbackBox.ts

```diff
@@ -50,7 +50,7 @@ const FeedbackBox = {
       borderTopWidth: "1px",
       color: "ui.typography.heading",
       display: "flex",
-      fontSize: "text.default",
+      fontSize: "desktop.body.body1",
```

Comment:
I don't notice a difference when this is removed.

Reply:
Interesting.  You don't see a change when removing the `fontSize` style altogether?  Or you don't see a change when using the original style (text.default)?